scripts/create.ts

```diff
@@ -109,7 +108,7 @@ process.stdout.write(`${COLORS.Reset}`);
   }
 
   // determines how many `../` are needed for package contents
-  const numDirectoriesAwayFromRoot = 3 + location.split(sep).length;
+  const numDirectoriesAwayFromRoot = 2 + location.split(sep).length;
```

Comment:
I only changed it because the previous version didn't support Window's path separators. I probably should have tested my change before committing. :ha: :grimacing: